### ISSUE TITLE
altered letters and numbers

### ISSUE THREAD

**Issue by cjarmada** [mdn/yari#6152]

**Comment by wbamberg** [comment]
Great! Since this is a platform issue, I'm transferring it to mdn/yari.

**Comment by cjarmada** [comment]
![Screenshot (83)](https://user-images.githubusercontent.com/97320757/166391210-7c201cb4-d0f7-4bc2-b38e-7629c439f98a.png)

And if I refresh the page, it's just fixed.

**Comment by caugner** [comment]
> * open a page, like say https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice
> 
> * from the sidebar, "open in new tab" a new page, like https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/copyWithin, so it opens in the background
> 
> * switch to that new tab

Unfortunately I cannot reproduce the issue following those steps with Chrome 101 on Mac OS.

@cjarmada @wbamberg 

- What Chrome version and operating system are you using?
- Could you please provide a screen recording (video) starting with a blank page (`about:blank`) so that we can see exactly what steps lead to the issue?

**Comment by caugner** [comment]
Duplicate of https://github.com/mdn/bob/issues/742.

**Comment by wbamberg** [comment]
I'm running macOS Big Sur, and see the problem with Chrome 100 (but since this has been an issue for years, it's unlikely to be a problem with a specific Chrome version.

Screen recording:

![overlaps](https://user-images.githubusercontent.com/432915/166489567-cdf3d55b-d776-4d8f-9ae1-67ce43c50e91.gif)

**Comment by caugner** [comment]
Thanks @wbamberg! I was able to reproduce it now, but only after disabling MDN Offline.

It looks like a CodeMirror bug, which doesn't set the `width` of the `div.CodeMirror-linenumber`:

<img width="1897" alt="image" src="https://user-images.githubusercontent.com/495429/166589959-695c2197-a445-49f2-a0c0-4a3fffa7f8eb.png">

And there's indeed a corresponding upstream bug: https://github.com/codemirror/CodeMirror/issues/5876

**Comment by schockocraft** [comment]
I am using this script in Tampermonkey now, to fix the issue on my end, and thought it might be worth suggesting that you'll implement the same workaround for everyone until the upstream issues have been resolved:

https://gist.github.com/schockocraft/490e9712ebdd9826165df86f1b128d22
